docs/data/rpc/api-reference/methods/sendTransaction.mdx

```diff
@@ -4,5 +4,143 @@ description: Submits a transaction
+
+<div>
+<CodeExample>
+
+```python
+# pip install --upgrade stellar-sdk
+from stellar_sdk import SorobanServer, soroban_rpc, Account, Asset, Keypair, Network, TransactionBuilder
+
+def send_transaction() -> soroban_rpc.SendTransactionResponse:
+    server = SorobanServer(server_url='https://soroban-testnet.stellar.org', client=None)
+
+    root_keypair = Keypair.from_secret(
+        "SA6XHAH4GNLRWWWF6TEVEWNS44CBNFAJWHWOPZCVZOUXSQA7BOYN7XHC"
+    )
+    # Create an Account object from an address and sequence number.
+    root_account = Account(account=root_keypair.public_key, sequence=1)
+
+    transaction = (
+        TransactionBuilder(
+            source_account=root_account,
+            # If you want to submit to pubnet, you need to change `network_passphrase` to `Network.PUBLIC_NETWORK_PASSPHRASE`
+            network_passphrase=Network.TESTNET_NETWORK_PASSPHRASE,
+            base_fee=100,
+        )
+        .append_payment_op(  # add a payment operation to the transaction
+            destination="GASOCNHNNLYFNMDJYQ3XFMI7BYHIOCFW3GJEOWRPEGK2TDPGTG2E5EDW",
+            asset=Asset.native(),
+            amount="125.5",
+        )
+        .append_set_options_op(  # add a set options operation to the transaction
+            home_domain="overcat.me"
+        )
+        .set_timeout(30)
+        .build()
+    )  # mark this transaction as valid only for the next 30 seconds
+    response = server.send_transaction(transaction)
+    return response
+
+response = send_transaction()
+
+print("response", response)
+```
+
+```js
+// yarn add @stellar/stellar-sdk
+import * as StellarSdk from "@stellar/stellar-sdk";
+
+import { Server } from "@stellar/stellar-sdk/rpc";
+const server = new Server("https://soroban-testnet.stellar.org");
+
+async function sendTransaction() {
+  try {
+    const contractId =
+      "CA3D5KRYM6CB7OWQ6TWYRR3Z4T7GNZLKERYNZGGA5SOAOPIFY6YQGAXE";
+    const contract = new StellarSdk.Contract(contractId);
+
+    const fee = StellarSdk.BASE_FEE;
+    const transaction = new StellarSdk.TransactionBuilder(account, { fee })
+      //.setNetworkPassphrase(StellarSdk.Networks.PUBLIC)
+      .setNetworkPassphrase(StellarSdk.Networks.FUTURENET)
+      .setTimeout(30) // valid for the next 30s
+      // Add an operation to call increment() on the contract
+      .addOperation(contract.call("increment"))
+      .build();
+
+    const sourceKeypair = StellarSdk.Keypair.fromSecret(sourceSecretKey);
+    transaction.sign(sourceKeypair);
+
+    server.sendTransaction(transaction).then((result) => {
+      console.log("hash:", result.hash);
+      console.log("status:", result.status);
+      console.log("errorResultXdr:", result.errorResultXdr);
+    });
+  } catch (error) {
+    console.error("Error fetching transaction:", error);
+  }
+}
+
+sendTransaction();
+```
+
+```java
+// implementation 'network.lightsail:stellar-sdk:0.44.0'
```

Comment:
I'm not very familiar with Java, so this might be a stupid question: Is this the equivalent to `pip install ...` or `yarn add ...`?

Reply:
yes this is showing that this library is needed to run the code